lib/vagrant-cookbook-fetcher/action_set_chef_paths.rb

```diff
@@ -8,8 +8,8 @@ def initialize(app, env)
       
       def call(env)
         # there has got to be a better way
-        provisioners_list = env[:machine].config.vm.provisioners
-        chef_solo = provisioners_list.find { |p| p.name === :chef_solo }
+        key = ::Gem::Specification::find_by_name('vagrant').version < Gem::Version.new('1.7.0') ? :name : :type
```

Comment:
After looking into it a little more there doesn't seem to be a standard way to require a minimum vagrant version, other than mention it in the Readme